CMakeLists.txt

```diff
@@ -31,7 +31,7 @@
 # the build configuration options are initialized.                                 #
 ####################################################################################
 
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.10)
```

Comment:
3.10 version is too strict, IMO 3.0 is enough, unless you use something specific.

Reply:
thanks!
ok, cmake version 3.0 is fine for me, I rather work with the newest, since there is better behavior :-)

Reply:
and it is not a problem for the user to have higher version